src/collaboration/membershipsUtils.ts

```diff
@@ -29,26 +30,54 @@ type MembershipQuery = Partial<
     CollaborationMembershipsPendingFragment
 >;
 
+export type MembershipsInfo = {
+  totalCount?: number;
+  pendingCount?: number;
+  accountMembership?: Membership;
+  membershipsSample?: Membership[];
+  enrollMethod?: string;
+  membershipType?: string;
+};
+
+export type MembershipList = {
+  // TODO: massage membershipsUtils/Service so more of these can be required
```

Comment:
@josebui do you want to complete this TODO now?

Reply:
@paulschreiber No, I think all these changes are big enough to stop here for now

Reply:
Please write up an issue to track this work.